docker/opbeans/workload/requirements.txt

```diff
@@ -0,0 +1,12 @@
+honcho==1.0.1
+molotov==1.6
+uvloop==0.11.0
+
```

Comment:
does this space do something? just curious

Reply:
just trying to separate direct dependencies from their sub-dependencies